cmake/CMakeLists.txt

```diff
@@ -169,6 +169,9 @@ option(onnxruntime_ENABLE_BITCODE "Enable bitcode for iOS only" OFF)
 # build eager mode
 option(onnxruntime_ENABLE_EAGER_MODE "build ort eager mode")
 
+# build Pytorch's LazyTensor support
+option(onnxruntime_ENABLE_LAZY_TENSOR "build custom JIT engine for Pytorch")
```

Comment:
Is it for training only?

Reply:
It's training-only. This PR registers ORT as a Pytorch backend. In inference, we usually don't have Pytorch so this feature won't be used.

Reply:
See #12630 . I suggest making it more explicitly.  For example, use https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html to hide it when training is not enabled. Or, at least add a comment to say this is training only.  It would save the other developers tons of time.